### PR TITLE
Restore grid

### PR DESCRIPTION
**Pull request by mardy**

### CODE REVIEW

**Comment by cibersheep** [comment]
I would say: don't go back.
Settings as a list are more clear to use and fits better with the over all system look

**Comment by mardy** [comment]
I've developed and tested this on a BQ E4.5; it would be nice if someone else can test it on more devices. Built packages are [here](https://ci.ubports.com/blue/organizations/jenkins/system-settings/detail/PR-198/2/artifacts).

**Comment by mardy** [comment]
> I would say: don't go back.

Let's discuss whether we want it in https://github.com/ubports/ubuntu-touch/issues/324, and only talk about the code here.

File: src/qml/CategorySection.qml [hunk 1]
```diff
@@ -22,6 +21,7 @@ import QtQuick 2.4
 import SystemSettings 1.0
 import SystemSettings.ListItems 1.0 as SettingsListItems
 import Ubuntu.Components 1.3
+import Ubuntu.Components.ListItems 1.3 as ListItem
```
Comment (cibersheep) [inline src/qml/CategorySection.qml]: ListItems is [deprecated](https://api-docs.ubports.com/sdk/apps/qml/Ubuntu.Components.ListItems/Base.html#detailed-description). Please, don't use them in new code, instead use ListItem

File: src/qml/EntryComponentList.qml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ * WITHOUT ANY WARRANTY; without even the implied warranties of
+ * MERCHANTABILITY, SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR
+ * PURPOSE.  See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import QtQuick 2.4
+import SystemSettings.ListItems 1.0 as SettingsListItems
```
Comment (cibersheep) [inline src/qml/EntryComponentList.qml:21]: ListItems is [deprecated](https://api-docs.ubports.com/sdk/apps/qml/Ubuntu.Components.ListItems/Base.html#detailed-description). Please, don't use them in new code, instead use ListItem

File: src/qml/CategorySection.qml (hunk 1 above)
Comment (mardy) [reply src/qml/CategorySection.qml]: OK, will change.

File: src/qml/EntryComponentList.qml (hunk 2 above)
Comment (mardy) [reply src/qml/EntryComponentList.qml:21]: No, these are not the UITK's listitems, but some special classes used in SystemSettings (note the import line). And internally they use the new ListItem.

Comment (cibersheep) [reply src/qml/EntryComponentList.qml:21]: Oh, sorry.

File: src/qml/CategorySection.qml (hunk 1 above)
Comment (cibersheep) [reply src/qml/CategorySection.qml]: Thank you

Comment (mardy) [reply src/qml/CategorySection.qml]: I pushed a commit, to fix this, but I don't have the device with me right now, so you'd have to wait a couple of days, unless someone is able to test it before me.
Till then, please don't merge this (though I assume there's no risk of that happening in such a hurry :-) ).

**Review by UniversalSuperBox — APPROVED** [review]
I didn't know how much I missed this.

**Pull request merged**